docs/doc/10-deploy/02-deploying-with-cloud-object-storage.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Deploying with Cloud Object Storage
+sidebar_label: Deploying with Cloud Object Storage
+description:
+  How to deploy Databend with cloud object storage
+---
+
+## Deploying with Cloud Object Storage
```

Comment:
How about `Deploying with S3 Compatible Storage Services`

Reply:
Hello @Xuanwo 

Thanks for the review. This section is intended for all the cloud storage services, azure blob, and s3 compatible ones. 
I just removed the s3 from the understanding deployment section (text and images) to reflect that Databend supports all types of cloud storage solutions.